discovery-infra/start_discovery.py

```diff
@@ -22,6 +23,18 @@
 import bootstrap_in_place as ibip
 
 
+# Any change here should be reflected in scripts/haproxy.cfg as well
+IMAGE_REGISTRIES = [
+    "quay.io",
+    "cdn02.quay.io",
+    "mirror.openshift.com",
+    "registry.ci.openshift.org",
+    "ci-dv2np-image-registry-us-east-1-aunteqmixxpqypvdqwbmjbiloeix.s3.dualstack.us-east-1.amazonaws.com",
+    "registry.build02.ci.openshift.org",
+]
+
+REVERSE_PROXY_IP = "1001:db8::1"
```

Comment:
Can we calculate according to the network we use? Not just hard coded network?

Reply:
same

Reply:
on the way class MachineNetwork too :)

Reply:
I hope you appreciate the lua code you made me write